pkgs/data/fonts/scientifica/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "scientifica";
+  version = "v2.1";
```

Versions in nix do not start with the v letter that GitHub suggests to use for versions.

```suggestion
  version = "2.1";
```